tests/src/Interop/ICustomMarshaler/MultipleALCs/CustomMarshaler.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+public class WrappedString
+{
+    public WrappedString(string str)
+    {
+        _str = str;
+    }
+
+    internal string _str;
+}
+
+public class WrappedStringCustomMarshaler : ICustomMarshaler
+{
+    private static readonly WrappedStringCustomMarshaler instance = new WrappedStringCustomMarshaler();
```

Comment:
@jeffschwMSFT Had mentioned that static member isolation might be broken in ALCs.